src/Microsoft.TestPlatform.ObjectModel/Constants.cs

```diff
@@ -110,6 +110,8 @@ public static class Constants
 
         public const string DotNetFrameWorkStringPrefix = ".NETFramework,Version=";
 
+        public const string DotNetFramework35 = ".NETFramework,Version = v3.5";
```

Comment:
Is it correct to have white spaces in between? Other platform versions don't have space.